docs/utilities/data_classes.md

```diff
@@ -119,25 +123,24 @@ Use **`APIGatewayAuthorizerRequestEvent`** for type `REQUEST` and **`APIGatewayA
     def handler(event: APIGatewayAuthorizerRequestEvent, context):
         user = get_user_by_token(event.get_header_value("Authorization"))
 
+        if user is None:
+            # No user was found, so we raised a not authorized error
+            raise UnAuthorizedError("Not authorized to perform this action")
```

Comment:
We need to test this with API GW. I don't think API GW is smart enough to recognise and return 403 - it'll throw a 500 back IIRC.

Reply:
This will also pollute Lambda metrics with errors, also causing cold starts since it's not handled.

We can make this work with event handler by catching it but not here I'm afraid